lib/Entities/StatisticsClickEntity.php

```diff
@@ -73,4 +73,39 @@ public function getLink() {
     $this->safelyLoadToOneAssociation('link');
     return $this->link;
   }
+
+  /**
+   * @param NewsletterEntity|null $newsletter
+   */
+  public function setNewsletter($newsletter) {
+    $this->newsletter = $newsletter;
+  }
+
+  /**
+   * @param SendingQueueEntity|null $queue
+   */
+  public function setQueue($queue) {
+    $this->queue = $queue;
+  }
+
+  /**
+   * @param int|null $subscriberId
+   */
+  public function setSubscriberId($subscriberId) {
+    $this->subscriberId = $subscriberId;
+  }
+
+  /**
+   * @param NewsletterLinkEntity|null $link
+   */
+  public function setLink($link) {
+    $this->link = $link;
+  }
```

Comment:
All of these are in fact non-nullable in the DB so I think it would make sense to use constructor (in all of the statistic entities). (They are nullable on PHP side but only due to `safelyLoadToOneAssociation`.)